c++/src/sargs/PredicateLeaf.cc

```diff
@@ -701,24 +707,22 @@ namespace orc {
       }
     }
 
-    // files written by trino may lack of hasnull field.
-    if (!colStats.has_has_null()) return TruthValue::YES_NO_NULL;
```

Comment:
Is it better to keep this line so lines below do not need to change?

Reply:
It seems better